packages/broker/src/plugins/operator/OperatorPlugin.ts

```diff
@@ -45,10 +48,20 @@ export class OperatorPlugin extends Plugin<OperatorPluginConfig> {
             this.streamrClient,
             toEthereumAddress(this.pluginConfig.operatorContractAddress)
         )
+        const operatorFleetState = new OperatorFleetState(
+            this.streamrClient,
+            toStreamID('/operator/coordination', serviceHelperConfig.operatorContractAddress)
+        )
         this.maintainTopologyService = new MaintainTopologyService(
             this.streamrClient,
-            new MaintainTopologyHelper(
-                serviceHelperConfig
+            new StreamAssignmentLoadBalancer(
+                'myNodeId', // TODO: real nodeId here
```

Comment:
Should maybe use `setUpAndStartMaintainTopologyService` here but difficulty due to asynchronicity.